devops/ci/multi_node_deploy/3node_deploy.jenkinsfile

```diff
@@ -112,16 +112,16 @@ node ('provisioning'){
                     echo \"\"\"
                     [cluster]
                     mgmt_vip=${MGMT_VIP}
+                    durability.storage-set-1.sns.data=4
```

Comment:
Please handle this internally without changing anything in config.ini

Reply:
Earlier it was handling it in storage_set[0] feild and now with this change it would make it modular.
reverting the change as it would be a overhead to test and merge it.